.eslintrc.js

```diff
@@ -168,5 +168,4 @@ module.exports = {
       typescript: {}, // this loads <rootdir>/tsconfig.json to eslint
     },
   },
-  ignorePatterns: ['src/params.js'],
```

Comment:
have to put this into `.eslintignore` since my vscode eslint doesn't like this somehow and it breaks my local eslint 🤷‍♂️